datahub-graphql-core/src/main/resources/timeline.graphql

```diff
@@ -0,0 +1,146 @@
+extend type Query {
+    """
+    Get list of schema blame
+    """
+    listSchemaBlame(input: ListSchemaBlameInput!): ListSchemaBlameResult
+}
+
+type TimelineConfig {
+    """
+    Whether the timeline feature is enabled.
+    """
+    enabled: Boolean!
+}
+
+"""
+Enum of SemanticVersionChange
+"""
+enum SemanticVersionChangeType {
+    NONE
+    PATCH
+    MAJOR
+    MINOR
+    EXCEPTIONAL
+}
+
+
+"""
+Enum of types of changes
+"""
+enum ChangeOperationType {
+    ADD
+    MODIFY
+    REMOVE
+}
+
+"""
+Enum of CategoryTypes
+"""
+enum ChangeCategoryType {
+    DOCUMENTATION
+    GLOSSARY_TERM
+    OWNERSHIP
+    TECHNICAL_SCHEMA
+    TAG
+}
+
+"""
+Input for getting list of latest schema changes computed at a specific version.
+"""
+input ListSchemaBlameInput {
+    """
+    The dataset urn
+    """
+    datasetUrn: String!
+    """
+    Categories to filter on
+    """
+    categories: [ChangeCategoryType!]
+    """
+    Changes after version cutoff are not shown
+    """
+    versionCutoff: String
+
+}
+
+"""
+List of latest schema changes computed at a specific version.
+"""
+type ListSchemaBlameResult {
+    """
+    Latest and current semantic version
+    """
+    latestSemVer: SemVerStruct!
+    """
+    Selected semantic version
+    """
+    selectedSemVer: SemVerStruct
+    """
+    All semantic versions
+    """
+    semVerList: [SemVerStruct!]
+    """
+    List of schema histories
+    """
+    schemaBlameList: [SchemaBlame!]
+}
+
+"""
+Schema history
+"""
+type SchemaBlame {
+    """
+    Flattened name of a field identifying the field the editable info is applied to
+    """
+    fieldPath: String!
+    """
+    Schema change
+    """
+    schemaChange : SchemaChange
+}
+
+type SchemaChange {
+    """
+    The time at which the schema was updated
+    """
+    timestampMillis: Long!
```

Comment:
Should ChangeCategoryType also be in here? Where is that reflected?

Reply:
It's reflected in the description which is created in the resolver, not on the frontend